.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 .env
 .botrc
 .hubot_history
+.idea
```

Comment:
Que?

(If you're using an IDE: it's better to set up a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for things like this. Arguably I should do that for .DS_Store as well, but I think that came from the generator, so meh.)